llama_stack/providers/utils/memory/vector_store.py

```diff
@@ -92,7 +92,24 @@ def content_from_data_and_mime_type(data: bytes | str, mime_type: str | None, en
     mime_category = mime_type.split("/")[0] if mime_type else None
     if mime_category == "text":
         # For text-based files (including CSV, MD)
-        return data.decode(encoding)
+        try:
+            return data.decode(encoding)
+        except Exception as original_error:
```

Comment:
I thought we were trying to catch `UnicodeDecodeError`?

Reply:
done